pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:leak_tracker/leak_tracker.dart';
+import 'package:meta/meta.dart';
+
+void _emptyLeakHandler(Leaks leaks) {}
+
+/// Leak tracking settings and helper methods for tests.
+abstract class LeakTrackingForTests {
+  /// Current configuration for leak tracking.
+  static LeakTrackingForTestsSettings settings =
+      const LeakTrackingForTestsSettings();
+
+  /// Updates [settings] to pause leak tracking.
+  static void pause() => settings = paused();
+
+  /// Updates [settings] to start leak tracking.
+  static void start() => settings = started();
+
+  /// Creates a copy of current settings with [paused] set to true.
+  static LeakTrackingForTestsSettings paused() =>
```

Comment:
This API is a bit confusing. It is not clear to me as a user what the difference between calling `pause` and `paused` is supposed to be.

Reply:
Comment for the class should help.

Reply:
And `paused` is renamed to `ignore` now.